README.md

```diff
@@ -4,8 +4,64 @@ This is a [kubernetes addon](https://github.com/mesosphere/kubeaddons) repositor
 
 # Using and Releasing `kubeaddons-kommander` Repository
 
-This repository follows the exact same rules that [`kubernetes-base-addons`](https://github.com/mesosphere/kubernetes-base-addons) does, please check documentation over there.
+This repository mostly follows the same rules that [`kubernetes-base-addons`](https://github.com/mesosphere/kubernetes-base-addons) does. Please familiarize yourself with that documentation, and then see the differences below.
 
 - [README.md](https://github.com/mesosphere/kubernetes-base-addons/blob/master/README.md)
 - [RELEASE.md](https://github.com/mesosphere/kubernetes-base-addons/blob/master/RELEASE.md)
 - [CONTRIBUTING.md](https://github.com/mesosphere/kubernetes-base-addons/blob/master/CONTRIBUTING.md)
+
+## Deviations from `kubernetes-base-addons` release process
+
+Kommander being a slightly different type of repository than `kubernetes-base-addons`, some details in releasing kommander are different.
+
+- Pre-releases are only tagged with `testing-` tags, no `stable-` tags for pre-releases. The `-rc` that will acutally be the stable GA release will be _also_ tagged with `stable-` tag (and w/o `-rc` suffix)
+- Kommander follows a more semver style versioning and is setup to be able to support multiple major and minor versions at the same time.
+
+## Step by step guides to release Kommander versions (pre-releases, latest and previously released)
+
+Only certain people are allowed to push to `testing` and `stable` branches. If you are asked to do a release, but dont have permissions to push, please ask your manager.
+
+### Tag/release pre-releases and update update SOAK
+
+1. fetch latest repo state: `git fetch`
+1. checkout `testing` branch: `git checkout testing`
+1. _merge_ `origin/master` into `testing` and push updated `testing` branch: `git merge origin/master && git push`
+1. apply tags, kommander uses multiple tags at this point:
+   1. apply and push "base" semver based tag: e.g. `git tag v1.1.0-beta.3 && git push origin v1.1.0-beta.3`
+   1. apply and push "consumable" testing tag(s) for each supported k8s version: e.g. `git tag testing-1.16-1.1.0-beta.3 && git push origin testing-1.16-1.1.0-beta.3`
+1. head to github and update release information for that prerelease: check [releases page](https://github.com/mesosphere/kubeaddons-kommander/releases) for up-to-date example. _Make sure to edit the "base" release_.
+1. make sure SOAK gets updated, you can sync with people who tag `kubernetes-base-addons`.
```

Comment:
Should we not include instructions to update soak? Isn't it updating the cluster.yaml and `konvoy deploy addons`?

Or is it because this is a public repo and we don't want to reference private ones?

Reply:
releasing kommander and updating soak are two different things, I'd think there is descriptions about how to update soak in soak repo 🤷